sdk/core/core-amqp/src/connectionConfig/connectionConfig.ts

```diff
@@ -134,7 +134,8 @@ export const ConnectionConfig = {
     if (options.isEntityPathRequired && !config.entityPath) {
       throw new TypeError("Missing 'entityPath' in configuration");
     }
-    config.entityPath = String(config.entityPath);
+    config.entityPath =
+      config.entityPath == undefined ? config.entityPath : String(config.entityPath);
```

Comment:
Nit: Unnecessary assignment?

```suggestion
    if (config.entityPath != undefined) {
       config.entityPath = String(config.entityPath);
    }
```